src/lucky_migrator/create_table_statement.cr

```diff
@@ -87,17 +87,18 @@ class LuckyMigrator::CreateTableStatement
     {% end %}
 
     {% if index || unique %}
-      add_index "{{ type_declaration.var }}", using: {{ using }}, unique: {{ unique }}
+      add_index :{{ type_declaration.var }}, using: {{ using }}, unique: {{ unique }}
     {% end %}
   end
 
-  def add_column(name, type : (String | Time | Int32 | Int64 | Float | Bool).class, optional = false)
+  def add_column(name, type : (String | Time | Int32 | Int64 | Float | Bool).class, optional = false, reference = "")
```

Comment:
I think `references` default should be `nil` instead. I think it will make it more clear what's going on